docs-user/guide-stack-samples-and-call-trees.md

```diff
@@ -36,7 +37,7 @@ In summary, self time is where work was actually happening when observed by the
 
 > Note: Feel free to follow along with a real call tree that reproduces this structure: [https://perfht.ml/2w45IdC](https://perfht.ml/2w45IdC)
 
-The simple call tree above did not take into account any branching stacks. It only concerned itself with running and self time. This section will dive into how the call tree handles stacks of different shapes.
+The uncomplicated call tree above did not take into account any branching stacks. It only concerned itself with running and self time. This section will dive into how the call tree handles stacks of different shapes.
```

Comment:
Hm, not sure how I feel about the new version here. I feel like it's not making it any better. Maybe we can keep the old version with an ignore?

Reply:
yeah, why not